release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -80,6 +80,11 @@ This release introduces the following updates to the *Developer* perspective of
 [id="ocp-4-16-openshift-cli"]
 === OpenShift CLI (oc)
 
+[id="ocp-4-16-oc-adm-upgrade-status-tp"]
+==== Introducing `oc adm upgrade status` command (Tech Preview)
```

Comment:
Also I think in our docs the spelled out term should be Technology Preview, I think tech preview is sort of informal/internal jargon:
```suggestion
==== Introducing `oc adm upgrade status` command (Technology Preview)
```